lib/plurimath/math/function/binary_function.rb

```diff
@@ -11,6 +11,16 @@ def initialize(parameter_one = nil, parameter_two = nil)
           @parameter_two = parameter_two
         end
 
+        def to_asciimath
+          "#{self.class.name.split('::').last.downcase}#{value_to_asciimath}"
+        end
```

Comment:
It seems to me that this method is currently not being used anywhere because of the method overriding in each class that inherits from `BinaryFunction`.

```ruby
first_value = "_(#{parameter_one.to_asciimath})" if parameter_one
second_value = "^(#{parameter_two.to_asciimath})" if parameter_two
"int#{first_value}#{second_value}"
```

It looks like the above code is repeated many times in binary classes. Can we move that here and only add the `to_asciimath` method where the pattern is different.

Reply:
> It seems to me that this method is currently not being used anywhere because of the method overriding in each class that inherits from `BinaryFunction`.
> 
> ```ruby
> first_value = "_(#{parameter_one.to_asciimath})" if parameter_one
> second_value = "^(#{parameter_two.to_asciimath})" if parameter_two
> "int#{first_value}#{second_value}"
> ```
> 
> It looks like the above code is repeated many times in binary classes. Can we move that here and only add the `to_asciimath` method where the pattern is different.

The ``to_asciimath`` method in **``BinaryFunction``** class is being used more than the methods defined in each class.
It's used in other files which had no changes and are not included in this PR.